scripts/util.sh

```diff
@@ -94,17 +94,19 @@ createKsApp() {
   ks generate openvino openvino
   ks generate jupyter jupyter
   ks generate notebook-controller notebook-controller
-  ks generate jupyter-web-app jupyter-web-app
-  ks generate centraldashboard centraldashboard
-  ks generate tf-job-operator tf-job-operator
-  ks generate tensorboard tensorboard
+  ks generate jupyter-web-app jupyter-web-app --injectIstio false
```

Comment:
this is used by kfctl.sh right?
Do you need to update anything for kfctl golang?

Reply:
no, because we stopped maintaining kfctl.sh, we don't have an option in kfctl.sh to install Istio.  this is why I disable `injectIstio` on all components.  golang is fine since we default to install Istio now.